server/src/utils/preprocessing/precinct_parser.py

```diff
@@ -34,7 +34,6 @@ def parse_precinct_table() -> pd.DataFrame:
         mapped_names = {
             "votdst": "precinct_code",
             "NAME": "precinct_name",
```

Comment:
So by default that column will be already be called geometry; I'm just not overriding the default name. You'll notice I originally had it map from the name "geometry" to "shape". That wasn't anything technical, just preference, but I later found out that geopandas (the library I'm using to store all this data), generally prefers it to be named "geometry". That makes it so that for many of the operations you don't have to explicitly specify the geometry column.